Microsoft.Identity.Web/Client/TokenCacheProviders/Session/MSALPerUserSessionTokenCacheProvider.cs

```diff
@@ -24,27 +24,31 @@
 
 using Microsoft.AspNetCore.Http;
 using Microsoft.Identity.Client;
+using System;
 using System.Diagnostics;
 using System.Security.Claims;
 using System.Threading;
 
 namespace Microsoft.Identity.Web.Client.TokenCacheProviders
 {
     /// <summary>
-    /// This is a MSAL's TokenCache implementation for one user. It uses Http session as a backend store
+    /// This is a MSAL's TokenCache implementation for one user. It uses Http session as a back end store
     /// </summary>
     public class MSALPerUserSessionTokenCacheProvider : IMSALUserTokenCacheProvider
     {
         private static ReaderWriterLockSlim SessionLock = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
 
         /// <summary>
-        /// Once the user signes in, this will not be null and can be ontained via a call to ClaimsPrincipal.Current
+        /// Once the user signs in, this will not be null and can be obtained via a call to ClaimsPrincipal.Current
         /// </summary>
+        // brentsch - why is this internal?
```

Comment:
@kalyankrishna1 ?

Reply:
The assembly InternalsVisibleTo a test project that I used to develop the token cache implementations. I later moved to a Mock for Http, so we can change it back to private now.

---
In reply to: [271425950](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/79#discussion_r271425950) [](ancestors = 271425950)

Reply:
OK